assets/src/dashboard/components/cardGallery/components.js

```diff
@@ -111,7 +106,8 @@ export const ActiveCard = styled.div(
     width: ${width}px;
     overflow: hidden;
     height: ${containerHeight}px;
-    border: ${theme.DEPRECATED_THEME.borders.gray75};
+    border: 1px solid ${theme.colors.border.defaultNormal};
+    border-radius: ${theme.borders.radius.small};
```

Comment:
Looks like the active card should be bigger?

|Current|Designs|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/110033619-ddce0e80-7cf6-11eb-832c-d3d70075d92d.png)|![image](https://user-images.githubusercontent.com/22185279/110033653-e7f00d00-7cf6-11eb-8b9a-5b328ddf440f.png)|

Reply:
samsies as the above with grid resizing. I think this should be separate and follow up.